uwblueprint/community-fridge-kw
# feat: extract calendar component and refactor scheduling flow

## Brief description. What is this change? 

### [Scheduling Flow: Extract calendar component](https://www.notion.so/uwblueprintexecs/Scheduling-Flow-Extract-calendar-component-to-be-92f6e65be1fb478b91729a6741a71ce2)

## Implementation description. How did you make this change?
* Extracted calendar from input field
* Made form progressive (reveals one section at a time)
* Moved back/next buttons & disable feature added to "next"
* "End date" input field was changed to date picker
* Overall styling was tweaked to match Figma designs
* Note: Was unable to remove "," from Calendar and to only display two weeks -> limitation of the library used (sorry Stacy!)

## Steps to test
1. Click through the form to see if each question is revealed one after the other
2. Ensure all buttons are clickable ("end date" field should only appear if not one time donation)
4.  Cross-check both Desktop and Mobile view with Figma designs

Desktop View: https://user-images.githubusercontent.com/69697180/150065707-0ebb5bdc-8727-47d1-a2ab-bbcc6f5ffffa.mp4
* @hanlinc27 Inconsistency with radio buttons is apparent from 0:24-0:30

Mobile View: https://user-images.githubusercontent.com/69697180/150065486-d421a912-963d-42bb-a499-b04f58014fbb.mov

Update: Steps 2 and 3 are now refactored as well

## Checklist
